simulator-core/src/test/java/com/evbox/everon/ocpp/testutils/ocpp/OcppMockServer.java

```diff
@@ -1,13 +1,18 @@
-package com.evbox.everon.ocpp.simulator.mock;
+package com.evbox.everon.ocpp.testutils.ocpp;
```

Comment:
Please keep **mock** as the package name, as it refers to mocking mechanism and I suggest to create all dependant/support classes in sub-packages, such as: mock.expect.* or mock.matcher.*

Reply:
done